saleor/graphql/order/types.py

```diff
@@ -886,10 +893,12 @@ def calculate_price(listing: Optional[ShippingMethodChannelListing]):
         )
 
     @staticmethod
+    @permission_required(OrderPermissions.MANAGE_ORDERS)
     def resolve_shipping_price(root: models.Order, _info):
         return root.shipping_price
 
     @staticmethod
+    @permission_required(OrderPermissions.MANAGE_ORDERS)
```

Comment:
Do logged-in customers have this permission? I can easily imagine a use case where the shipping price or the shipping address is displayed on the order details screen for customers.